spec/itunes/volume_spec.rb

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+require 'spec_helper'
+require 'itunes-client'
+
+include Itunes
+
+describe Volume do
+  let(:volume) { Itunes::Player.volume }
+  let(:level) { Fixnum }
```

Comment:
hmm what exactly do you mean by that? what would you use instead of real numbers?